csharp/dotnet/security/audit/ldap-injection.yaml

```
@@ -29,7 +29,7 @@ rules:
   pattern-sources:
   - patterns:
     - focus-metavariable: $INPUT
-    - pattern-inside: $T $M($INPUT,...) {...}
+    - pattern-inside: $T $M(..,$INPUT,...) {...}
```

this ellipsis is incomplete
```suggestion
    - pattern-inside: $T $M(...,$INPUT,...) {...}
```